wire-ios/Tests/Mocks/SwiftMockConversation+ConversationCreation.swift

```diff
@@ -0,0 +1,44 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+extension SwiftMockConversation {
+
+    static func oneOnOneConversation(
+        otherUser: UserType = MockUser.mockUsers().first!
+    ) -> SwiftMockConversation {
+        let selfUser = (MockUser.mockSelf() as Any) as! ZMUser
```

Comment:
why having two casts?

```suggestion
        let selfUser = MockUser.mockSelf() as! ZMUser
```

Reply:
If we remove it we get this warning: Cast from 'MockUser?' to unrelated type 'ZMUser' always fails

That's why I added it.